src/components/color-picker/tests/accessibility.e2e.ts

```diff
@@ -0,0 +1,332 @@
+import { E2EElement, E2EPage, newE2EPage } from "@stencil/core/testing";
+
+const defaultTranslations = {
+  accessibleName: {
+    lightnessChannelInput: "Lightness",
+    hueChannelInput: "Hue",
+    saturationChannelInput: "Saturation",
+    valueChannelInput: "Value",
+    alphaChannelInput: "Alpha",
+    redChannelInput: "Red",
+    greenChannelInput: "Green",
+    blueChannelInput: "Blue",
+    hexadecimalInput: "Hexadecimal color",
+    colorFormatOptions: "Color format options",
+    colorFieldControl: "Color field",
+    colorFieldDescription: "2d color field",
+    hueSliderControl: "Hue Slider",
+    alphaSliderControl: "Alpha Slider",
+    colorPaletteButton: "Selected color ${SELECTED_COLOR}",
+    currentColorPreview: "Current color: ${SELECTED_COLOR}"
```

Comment:
Are we missing the backsticks?

```suggestion
    colorPaletteButton: `Selected color ${SELECTED_COLOR}`,
    currentColorPreview: `Current color: ${SELECTED_COLOR}`
```